libs/functions/build_api_v2.py

```diff
@@ -73,6 +78,7 @@ def build_region_timeseries(
     for row in timeseries.yield_records():
         # Timeseries records don't have population
         row[CommonFields.POPULATION] = region_summary.population
+        row[CommonFields.LOCATION_ID] = region_summary.locationId
```

Comment:
I don't think we need to add this to the timeseries row.  Actually looking at it, I don't think we need population either (and don't believe it actually gets included in the actuals object).  I think you could remove population as well, but should at least remove location id here